modules/shapefile/src/lib/parsers/parse-shapefile.js

```diff
@@ -101,6 +106,19 @@ function joinProperties(geometries, properties) {
   return features;
 }
 
+/**
+ * Reproject GeoJSON features to output CRS
+ *
+ * @param  {object[]} features parsed GeoJSON features
+ * @param  {string} sourceCrs source coordinate reference system
+ * @param  {string} targetCrs †arget coordinate reference system
+ * @return {object[]} Reprojected Features
+ */
+function reprojectFeatures(features, sourceCrs, targetCrs) {
+  const projection = new Proj4Projection({from: sourceCrs || 'WGS84', to: targetCrs || 'WGS84'});
+  return transformGeoJsonCoords(features, coord => projection.project(coord));
```

Comment:
So `return transformGeoJsonCoords(features, projection.project);` does not work?

Reply:
Should work after https://github.com/uber-web/math.gl/pull/165 is merged and published.